README.md

```diff
@@ -2,51 +2,32 @@
 
 [![ipfs](http://img.shields.io/irc/%23ipfs.png)](https://webchat.freenode.net/?channels=ipfs)
 
-IPFS is a distributed file system that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single bittorrent swarm exchanging git objects.
+Welcome to IPFS (the InterPlanetary File System)! Why not [watch a video](https://www.youtube.com/watch?v=8CMxDNuuAiQ) to get started?
```

Comment:
Yep - don't "Bury the lead"

On Wed, Nov 25, 2015 at 12:30 PM, Juan Benet <notifications@github.com>
wrote:

> In README.md <https://github.com/ipfs/ipfs/pull/126#discussion_r45820006>:
>
> > @@ -2,51 +2,32 @@
> >
> >  [![ipfs](http://img.shields.io/irc/%23ipfs.png)](https://webchat.freenode.net/?channels=ipfs <http://img.shields.io/irc/%23ipfs.png)%5D(https://webchat.freenode.net/?channels=ipfs>)
> >
> > -IPFS is a distributed file system that seeks to connect all computing devices with the same system of files. In some ways, this is similar to the original aims of the Web, but IPFS is actually more similar to a single bittorrent swarm exchanging git objects.
> > +Welcome to IPFS (the InterPlanetary File System)! Why not [watch a video](https://www.youtube.com/watch?v=8CMxDNuuAiQ) to get started?
>
> I think we should lead with the single paragraph description. (same as
> should lead https://ipfs.io).
>
> we can *change* that single paragraph to be better, but it should be here
> i think.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ipfs/ipfs/pull/126/files#r45820006>.
>